config/settings/deployment.py

```diff
@@ -7,13 +7,13 @@
 from sentry_sdk.integrations.redis import RedisIntegration
 
 from .base import *  # noqa
-from .base import env
+from .base import APP_VERSION, DATABASES, TEMPLATES, env
```

Comment:
You need to import * from base right ? the configs in base are all needed in deployment.

Reply:
yes
the second import is to avoid having to ignore possibly undefined lint checks

Reply:
it will still import test of the config fron base

Reply:
nvm, my eyes aint working properly 😅